### DISCUSSION TITLE
Tech to use in Frontend

### DISCUSSION THREAD

**Discussion by umitcan07** [bounswe/bounswe2024group11#179] [Ideas]
- Typescript
- React
- Node (JS runtime)
- pnpm (Package Management)
- Mantine UI (UI Library and Core Styling)
- Tailwind CSS (Styling)
- Prettier (Code Formatting)
- ESLint (Code Quality)
-  React Router Dom (routing)

**Comment by hks1444** [comment]
Let’s use react router dom for routing.

**Comment by Meminseeker** [comment]
It seems like there is a big learning path for everyone. 'Kolay gelsin', as @furkanakkurt1335 said earlier

**Comment by umitcan07** [comment]
We realized pretteir + eslint will not be the best for us in terms of boring errors. We decided to omit them and use Biome instead. It covers the use case of prettier + eslint in one

**Reply by hks1444** [reply]
I agree. We had to waste time, which could be used to do more productive work, on those errors. It is better to use Biome.
